nixos/modules/services/networking/autossh.nix

```diff
@@ -0,0 +1,124 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.autossh;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.autossh = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = ''
+          Whether to enable the AutoSSH, the SSH sessions manager
+        '';
+      };
```

Comment:
@edolstra: I guess so, but isn't it nice that all services have a ```.enable```? Like a "common API"? services.phpfpm is a service that does not have a .enable option, it is automatically enabled if there is a config. I find it a bit confusing.

In https://github.com/NixOS/nixpkgs/pull/10551 I've opted for the .enable option, even though it _can_ be inferred from the other config.

Thoughts?